bridges/TwitterBridge.php

```diff
@@ -234,7 +234,31 @@ public function collectData(){
 
 			// generate the title
 			$item['title'] = $tweet->full_text;
-			$cleanedTweet  = $tweet->full_text;
+
+			// Convert plain text URLs into HTML hyperlinks
+			if (isset($tweet->entities->media)) {
+				foreach($tweet->entities->media as $media) {
+					$cleanedTweet = str_replace($media->url,
+						'<a href="' . $media->expanded_url . '">' . $media->display_url . '</a>',
+						$tweet->full_text);
+				}
+			} elseif (isset($tweet->entities->urls)) {
+				foreach($tweet->entities->urls as $url) {
+					$cleanedTweet = str_replace($url->url,
+						'<a href="' . $url->expanded_url . '">' . $url->display_url . '</a>',
+						$tweet->full_text);
+				}
+			} else {
+				// fallback to regex'es
+				$reg_ex = '/(http|https|ftp|ftps)\:\/\/[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,3}(\/\S*)?/';
```

Comment:
Maybe 't.co' links only? At least, not to conflict with suggestions above.

Reply:
Can we assume that they should always be in either of one of those $tweet->entities->media / $tweet->entities->urls ?

Reply:
Ok.